runtime/compiler/env/J9SharedCache.hpp

```diff
@@ -421,11 +422,14 @@ class TR_J9SharedCache : public TR_SharedCache
     /**
     * \brief Store the dependencies of an AOT method in the SCC
     *
-    * The dependencies of an AOT method are encoded as an array of uintptr_t values. The first entry is the number
-    * of elements in the entire array. The subsequent entries are encoded offsets to the class chains of classes that
-    * need to be loaded or initialized before the compiled body can be loaded. If the class must be initialized, the
-    * entry will be the offset itself (which necessarily has a set low bit), and if the class only needs to be loaded
-    * it will be the offset with the low bit cleared.    *
+    * The dependencies of an AOT method are encoded as an array of uintptr_t
+    * values. The first entry is the number of entries after the first, the
```

Comment:
The original documentation said the first entry was the total number of entries, but I actually had the dependencies set up similarly to the well-known classes chain, where the first entry is the number of subsequent entries.